configure.ac

```diff
@@ -975,6 +975,52 @@ AC_ARG_WITH(wasm,                [  --with-wasm=yes,no                  If you w
 
 
 AC_ARG_WITH(runtime_preset, [  --with-runtime_preset=net_4_x,all,aot,hybridaot,fullaot,bitcode,unreal   Which default profile to build (defaults to net_4_x)],  [], [with_runtime_preset=net_4_x])
+AC_ARG_WITH(spectre-mitigation,             [  --with-spectre-mitigation=yes,no   If you want to build the runtime with compiler flags that enable Meltdown/Spectre mitigation (defaults to no)], [], [with_spectre_mitigation=default])
```

Comment:
Isn't this mitigation only for Spectre, not Meltdown?

Reply:
It is, but I didn't want to split hairs here - no need to explain that meltdown is mitigated by separating kernel and userland page tables and the two exploits are mentioned in the same sentence all the time.

Reply:
I don't think that's splitting hairs TBH. This comment is basically saying that if you don't build with this option, Mono is vulnerable to Meltdown, which is false.

I don't think we should be propagating the Spectre/Meltdown confusion.